pkg/proxy/iptables/proxier.go

```diff
@@ -713,6 +715,9 @@ func (proxier *Proxier) syncProxyRules() {
 		if proxier.masqueradeAll {
 			writeLine(natRules, append(args, "-j", string(kubeMarkMasqChain))...)
 		}
+		if len(proxier.clusterCIDR) > 0 {
```

Comment:
This will write once per-service, won't it?  Can you achieve the same effect by writing once at ~L687